pythonforandroid/recommendations.py

```diff
@@ -191,8 +187,9 @@ def check_ndk_api(ndk_api, android_api):
 
 MIN_PYTHON_MAJOR_VERSION = 3
 MIN_PYTHON_MINOR_VERSION = 6
```

Comment:
Out of scope, but I'm wondering, is 3.6 still the minor recommended version?

Reply:
I'd encourage upgrading to Python 3.8 as a minimum, but I see that as a project decision.

* Python 3.6 and Python 3.7 are end of life. Python 3.8 has just under a year left of security support.
* The latest released version of Buildozer claims to support back to 3.6. The in-dev version is 3.8 minimum.
* The latest released version of Kivy claims to support back to 3.7. The in-dev version is 3.7 minimum.

* p4a's setup.py advertises 3.7, not 3.6.

My attitude is if you don't test it, you aren't really supporting it, so whatever the decision, please ensure the CI tests cover what `setup.py` advertises.

Reply:
About this specific issue (the one suggested by @AndreMiras ):
I will (immediately, so we do not miss it), open a new issue, and target with a separate proper PR.

Generally, about the minimum Python version:
- `kivy/kivy`: We still need to include `3.7` even if reached EOL, at least for `2.3.0` as `piwheels` still requires `3.7` wheels (due to debian buster). Then, I will check out with `piwheels` maintainer, if there's a chance to avoid distributing wheels for debian buster.
- Same applies for `kivy/pyobjus`, `kivy/pyjnius` and `kivy/plyer`, as are quite tied with `kivy/kivy`.
- For `kivy/python-for-android`, `kivy/kivy-ios` and `kivy/buildozer`: what are we waiting to set the minimum version to `3.8` ? 😅

Reply:
Great insight, thanks @misl6 
And yeah of course it makes sense to address that in dedicated PR, it caught my attention in this PR, but it's totally out of this scope